docs/key-issues/multiple_architectures.md

```diff
@@ -0,0 +1,259 @@
+# Platforms with multiple CPU architectures
+
+In addition to any ABI requirements, a binary is compiled for a CPU
+architecture. That CPU architecture defines the CPU instructions that can be
+issued by the binary.
+
+## Current state
+
+Historically, it could be assumed that an executable or library would be
+compiled for a single CPU archicture. On the rare occasion that an operating
+system was available for mulitple CPU architectures, it became the
+responsibility of the user to find (or compile) a binary that was compiled for
+their host CPU architecture.
+
+However, we now see operating system platforms where multiple CPU architectures
+are supported:
+
+* In the early days of Windows NT, both x86 and DEC Alpha CPUs were supported
+* Windows 10 supports x86, x86-64, ARMv7 and ARM64; Windows 11 supports x86-64
+  and ARM64.
+* Although Linux started as an x86 project, the Linux kernel is now available a
+  wide range of other CPU architectures, including ARM64, RISC-V, PowerPC, s390
+  and more.
```

Comment:
Can you think of an architecture (by any other vendor) which is _not_ supported by linux?

Reply:
Not off the top of my head. I'm sure one exists, but it's likely very niche and/or proprietary.

Reply:
My point was that this sells Linux quite a bit short - the "x86 project" started 30 years ago, and for the most part of its history, it has supported every architecture under the sun that has a larger-than-microscopic amount of users.

```suggestion
* Due to its open source nature, Linux tends to support all CPU architectures for
  which someone is interested enough to author & provide support in the kernel,
  see [here](https://en.wikipedia.org/wiki/List_of_Linux-supported_computer_architectures).
```

Reply:
I mean... sure. Not sure it advances the discussion much, but 🤷

Reply:
It's not supposed to advance the discussion, merely avoid arbitrarily volunteering falsehoods (which might result in taking the discussion *backwards*).

Reply:
> Not sure it advances the discussion much, but 🤷

From my side, much of the discussion has wrapped up, this was a minor remaining editorial point.